internal/access/identity.go

```diff
@@ -86,19 +86,25 @@ func ListIdentities(c *gin.Context, opts data.ListIdentityOptions) ([]models.Ide
 	return data.ListIdentities(db, opts)
 }
 
-func GetContextProviderIdentity(c RequestContext) (*models.Provider, string, error) {
+func GetContextProviderIdentity(c RequestContext, google *models.Provider) (*models.Provider, string, error) {
```

Comment:
should this parameter be `social *models.Provider` ?

Reply:
When we support more forms of social login we will have to update a bunch of areas to take a list of social login providers rather than one, so as long as its just Google I'll leave it explicit to make it easier to clean up later.